src/Analysis/GWASV2/GWASContainer.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState, useEffect } from "react";
+import { PropTypes } from "prop-types";
+import { Space, Button, Steps, Step, Popconfirm } from "antd";
+
+const GWASContainer = () => {
+    const [current, setCurrent] = useState(0);
+    const gwasSteps = [
+        {
+            title: 'Step 1',
+            description: 'Select Study Population',
+        },
+        {
+            title: 'Step 2',
+            description: 'Select Outcome Phenotype',
+        },
+        {
+            title: 'Step 3',
+            description: 'Select Covariate Phenotype',
+        },
+        {
+            title: 'Step 4',
+            description: 'Review and submit your GWAS study',
```

Comment:
Not sure where this new description is coming from. The wireframe seems to still state "Configure GWAS" https://xd.adobe.com/view/5773a0b3-0957-443b-830b-95318a30363c-0220/screen/b66dc7ca-840f-4cc9-92ba-0e02b21319fb

Reply:
is this the catch-call for all remaining things including but not limited to submission?

Reply:
edited

Reply:
yes...for now 😅